packages/core/src/routes/experience/classes/interaction-session.ts

```diff
@@ -91,7 +92,8 @@ export default class InteractionSession {
           status: 404,
         },
         {
-          identifier: verificationRecord.identifier.value,
+          identifier:
+            'identifier' in verificationRecord ? verificationRecord.identifier : undefined,
```

Comment:
This piece of logic will be refactored in the coming next PR, please ignore it. See the comments above.